homeassistant/core.py

```diff
@@ -260,6 +260,55 @@ def __str__(self) -> str:
         return self.value
 
 
+class BackgroundTasks:
+    """Class to manage background tasks."""
```

Comment:
Try to make the docstrings in this class more descriptive so it's clear when to use `hass.background_tasks.async_create_task` vs `hass.async_create_task`

Reply:
going to remove it instead.